script/Ciel/Day 3/03_01_CIEL03_3_3C.txt

```diff
@@ -91,22 +101,25 @@ She talked about school, various restaurants I didn't know about, and never let
 [sha:acf3c7f32a63c900ced7aa6915fe38d49c41e83c]{
 -- Page 250, Offset 5487.
 -- 満足な返答はひとつもできなかったけど、その<度|たび>にどうにかしていた頭が、少しずつ正常に戻っていくのが申し訳なく、同時に嬉しかった。
-I couldn't give her a single satisfactory reply, but I felt sorry and happy at the same time that my head was gradually returning to normal.
+I wasn't able to give even a single satisfactory reply,
+ but each time I did answer her I felt my mind gradually creep a little closer to normal.
+ It felt unforgivable, but at the same time, I was glad of it.
 }
 [sha:b8d91d6462675b400c0c24ec5f5626306a3b4d46]{
 -- Page 251, Offset 5488.
 -- ……本当に、なんてコトだろう。
-...Really, what a mess.
+I honestly can't believe it...
 }
 [sha:3f9aac956502c9ac6bf90387c931ef09f4e14100]{
 -- Page 251, Offset 5489.
 -- たった数時間前に人を殺しておいて、こんな普通の学生みたいな事にドギマギしてしまっている。
-Just a few hours ago I killed a person, and now I'm upset about something like this, like a normal student.
+It's only been a few hours since I killed a person,
+ and now here I am getting flustered over things like an ordinary high school student.
```

Comment:
sounds a bit like he's saying that he's getting flustered over 'things such as an ordinary hs student' rather than him acting like one.